diff --git a/node_modules/@theme-ui/color-modes/dist/declarations/src/index.d.ts b/node_modules/@theme-ui/color-modes/dist/declarations/src/index.d.ts
index 660c660..6e78eef 100644
--- a/node_modules/@theme-ui/color-modes/dist/declarations/src/index.d.ts
+++ b/node_modules/@theme-ui/color-modes/dist/declarations/src/index.d.ts
@@ -16,3 +16,5 @@ export declare const InitializeColorMode: () => React.DetailedReactHTMLElement<{
         __html: string;
     };
 }, HTMLElement>;
+export declare const createColorStyles: (theme?: import("@theme-ui/css").Theme) => import("@theme-ui/css").CSSObject;
+export declare const getPreferredColorScheme: () => "dark" | "light" | null
\ No newline at end of file
diff --git a/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.dev.js b/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.dev.js
index 51d10bf..14c161a 100644
--- a/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.dev.js
+++ b/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.dev.js
@@ -481,3 +481,5 @@ var InitializeColorMode = function InitializeColorMode() {
 exports.ColorModeProvider = ColorModeProvider;
 exports.InitializeColorMode = InitializeColorMode;
 exports.useColorMode = useColorMode;
+exports.createColorStyles = createColorStyles;
+exports.getPreferredColorScheme = getPreferredColorScheme;
diff --git a/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.prod.js b/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.prod.js
index 99c875c..19e6f4f 100644
--- a/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.prod.js
+++ b/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.prod.js
@@ -473,3 +473,5 @@ var InitializeColorMode = function InitializeColorMode() {
 exports.ColorModeProvider = ColorModeProvider;
 exports.InitializeColorMode = InitializeColorMode;
 exports.useColorMode = useColorMode;
+exports.createColorStyles = createColorStyles;
+exports.getPreferredColorScheme = getPreferredColorScheme;
diff --git a/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.esm.js b/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.esm.js
index e98a184..35bf0e2 100644
--- a/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.esm.js
+++ b/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.esm.js
@@ -470,4 +470,4 @@ var InitializeColorMode = function InitializeColorMode() {
   });
 };
 
-export { ColorModeProvider, InitializeColorMode, useColorMode };
+export { ColorModeProvider, InitializeColorMode, useColorMode, createColorStyles, getPreferredColorScheme };
