diff --git a/node_modules/@theme-ui/color-modes/dist/declarations/src/index.d.ts b/node_modules/@theme-ui/color-modes/dist/declarations/src/index.d.ts
index 660c660..479d0d5 100644
--- a/node_modules/@theme-ui/color-modes/dist/declarations/src/index.d.ts
+++ b/node_modules/@theme-ui/color-modes/dist/declarations/src/index.d.ts
@@ -16,3 +16,5 @@ export declare const InitializeColorMode: () => React.DetailedReactHTMLElement<{
         __html: string;
     };
 }, HTMLElement>;
+export declare const createColorStyles: (theme?: import("@theme-ui/css").Theme) => import("@theme-ui/css").CSSObject;
+export declare const getPreferredColorScheme: () => "dark" | "light" | null
diff --git a/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.dev.js b/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.dev.js
index dc6967a..8de9f0e 100644
--- a/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.dev.js
+++ b/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.dev.js
@@ -337,3 +337,5 @@ const InitializeColorMode = () => core.jsx('script', {
 exports.ColorModeProvider = ColorModeProvider;
 exports.InitializeColorMode = InitializeColorMode;
 exports.useColorMode = useColorMode;
+exports.createColorStyles = createColorStyles;
+exports.getPreferredColorScheme = getPreferredColorScheme;
diff --git a/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.prod.js b/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.prod.js
index 6ef2536..8dd1ecd 100644
--- a/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.prod.js
+++ b/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.cjs.prod.js
@@ -329,3 +329,5 @@ const InitializeColorMode = () => core.jsx('script', {
 exports.ColorModeProvider = ColorModeProvider;
 exports.InitializeColorMode = InitializeColorMode;
 exports.useColorMode = useColorMode;
+exports.createColorStyles = createColorStyles;
+exports.getPreferredColorScheme = getPreferredColorScheme;
diff --git a/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.esm.js b/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.esm.js
index 171481d..6bf3e9a 100644
--- a/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.esm.js
+++ b/node_modules/@theme-ui/color-modes/dist/theme-ui-color-modes.esm.js
@@ -326,4 +326,4 @@ const InitializeColorMode = () => jsx('script', {
   }
 });
 
-export { ColorModeProvider, InitializeColorMode, useColorMode };
+export { ColorModeProvider, InitializeColorMode, useColorMode, createColorStyles, getPreferredColorScheme };
