diff --git a/node_modules/next-images/index.js b/node_modules/next-images/index.js
index d06e5e1..9ac43da 100644
--- a/node_modules/next-images/index.js
+++ b/node_modules/next-images/index.js
@@ -52,7 +52,7 @@ module.exports = ({ dynamicAssetPrefix = false, ...nextConfig } = {}) => {
                       ''
                     }/_next/static/images/`,
                   }),
-              name: "[name]-[hash].[ext]",
+              name: nextConfig.name || "[name]-[hash].[ext]",
               esModule: nextConfig.esModule || false
             }
           }
