diff --git a/node_modules/next-page-tester/dist/testHelpers.js b/node_modules/next-page-tester/dist/testHelpers.js
index c680dcd..6c1c278 100644
--- a/node_modules/next-page-tester/dist/testHelpers.js
+++ b/node_modules/next-page-tester/dist/testHelpers.js
@@ -3,6 +3,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.cleanup = exports.initTestHelpers = void 0;
 const makeRenderMethods_1 = require("./makeRenderMethods");
 const setEnvVars_1 = require("./setEnvVars");
+const nonIsolatedModules = require("./utils").nonIsolatedModules;
 function isJSDOMEnvironment() {
     return navigator && navigator.userAgent.includes('jsdom');
 }
@@ -24,6 +25,14 @@ function initTestHelpers() {
     if (typeof document !== 'undefined' && typeof afterEach === 'function') {
         afterEach(cleanup);
     }
+    if (typeof jest !== "undefined") {
+        for (const moduleName of nonIsolatedModules) {
+            // @NOTE for some reason Jest needs us to pre-import the modules
+            // we want to require with jest.requireActual
+            require(moduleName);
+            jest.mock(moduleName, () => jest.requireActual(moduleName));
+        }
+    }
 }
 exports.initTestHelpers = initTestHelpers;
 function cleanup() {
diff --git a/node_modules/next-page-tester/dist/utils.js b/node_modules/next-page-tester/dist/utils.js
index 59e916c..019a7be 100644
--- a/node_modules/next-page-tester/dist/utils.js
+++ b/node_modules/next-page-tester/dist/utils.js
@@ -85,16 +85,11 @@ const nonIsolatedModules = [
     'next/dist/next-server/lib/router-context',
     'next/dist/next-server/lib/runtime-config',
 ];
+exports.nonIsolatedModules = nonIsolatedModules;
 function executeWithFreshModules(f) {
     /* istanbul ignore else */
     if (typeof jest !== 'undefined') {
         let result;
-        for (const moduleName of nonIsolatedModules) {
-            // @NOTE for some reason Jest needs us to pre-import the modules
-            // we want to require with jest.requireActual
-            require(moduleName);
-            jest.mock(moduleName, () => jest.requireActual(moduleName));
-        }
         jest.isolateModules(() => {
             result = f();
         });
